site/content/device/MF-RC522

```diff
@@ -1,7 +1,7 @@
 +++
 title = "MF-RC522"
-description = "1 bit OLED display controller"
-picture = "/img/ssd1306.jpg"
+description = "MIFARE and NTAG frontend"
+picture = "/img/mfrc522.JPG"
```

Comment:
Did you mean to git add the picture?

Reply:
I do not now, I just uploaded the picture in the same folder as the others. I never used git hub before and have no clue.

Reply:
No worry, the only thing you need to do is `git add` with the path of the image, the `git commit` then `git push`.